.scripts/select_wokwi_matrix.py

```
@@ -0,0 +1,192 @@
+#!/usr/bin/env python3
+"""Select Wokwi scenarios for CI based on event mode and changed files."""
+
+from __future__ import annotations
+
+import argparse
+import json
+import re
+from dataclasses import dataclass
+from pathlib import Path
+from typing import Iterable
```

Ruff (0.15.7)

[warning] Import from `collections.abc` instead: `Iterable`

Import from `collections.abc`

(UP035)